source/entities/Creature.cpp

```diff
@@ -296,10 +310,6 @@ void Creature::removeFromGameMap()
         home->releaseTileForSleeping(getHomeTile(), this);
     }
 
-    Tile* posTile = getPositionTile();
```

Comment:
No. That's because entities didn't use to be added on tiles on client side. Now, they are because we have to know which creature is on which tile to compute spell cost on client side.
Actually, that's the potential game breaker in this PR if I forgot to release something. But that's the good timing to do such things since we will have time to test and check for side effects